website/docs/cli/commands/vendor/vendor-pull.mdx

```diff
@@ -108,6 +108,124 @@ The following config can be used to download the `vpc` component from an AWS pub
+
+:::tip  
+In automated systems like GitHub Actions, we recommend sticking with the `https://` scheme for vendoring. Atmos will automatically inject the `GITHUB_TOKEN`.  
+:::
+
+There are two primary ways to specify an SSH source.
+
+### SCP-style Sources
+
+Atmos supports traditional SCP-style sources, which use a colon to separate the host from the repository, like this:
+
+```shell
+git::git@github.com:cloudposse/terraform-null-label.git?ref={{.Version}}
+```
+
+Atmos rewrites this URL to the following format:
+
+```shell
+git::ssh://git@github.com/cloudposse/terraform-null-label.git?depth=1&ref={{.Version}}
+```
+
+If no username is supplied and the host is `github.com`, Atmos automatically injects the default username `git`.
+
+### Explicit SSH Sources
+
+When the `ssh://` scheme is explicitly specified, the URL is used as provided, and no rewriting occurs.
+
+For example:
+
+```shell
+git::ssh://git@github.com/cloudposse/terraform-null-label.git?ref={{ .Version }}
+```
+
+
+### Important Notes
+
+- The following URL is **invalid** because `go-getter` misinterprets `github.com:` as a URL scheme (like `http:` or `git:`), causing a parsing error:
+  ```shell
+  github.com:cloudposse/terraform-null-label.git?ref={{ .Version }}
+  ```
+
+- When a URL has no scheme, Atmos defaults to **HTTPS** and injects credentials if available.
+  ```shell
+  github.com/cloudposse/terraform-null-label.git?ref={{ .Version }}
+  ```
+
+## Git over HTTPS Vendoring
+
+Atmos supports vendoring components using **Git over HTTPS**.
+
+For example:
+```
+github.com/cloudposse/terraform-null-label.git?ref={{ .Version }}
+```
+is automatically resolved as:
+```
+git::https://github.com/cloudposse/terraform-null-label.git?depth=1&ref={{ .Version }}
+```
+
+## Authentication & Token Usage for HTTPS
+
+Atmos prioritizes authentication credentials based on predefined environment variables. The priority order for each provider is:
+
+### GitHub
+<dl>
+ <dt>ATMOS_GITHUB_TOKEN</dt>
+ <dd>Bearer token for GitHub API requests, enabling authentication for private repositories and higher rate limits.</dd>
+
+ <dt>GITHUB_TOKEN</dt>
+ <dd>Used as a fallback if ATMOS_GITHUB_TOKEN is not set.</dd>
+</dl>
+
+**Default Username for HTTPS:** `"x-access-token"`
+
+### Bitbucket
+<dl>
+ <dt>ATMOS_BITBUCKET_TOKEN</dt>
+ <dd>Bitbucket app password for API requests; used to avoid rate limits. When both ATMOS_BITBUCKET_TOKEN and BITBUCKET_TOKEN are defined, the former prevails.</dd>
+
+ <dt>BITBUCKET_TOKEN</dt>
+ <dd>Used as a fallback when ATMOS_BITBUCKET_TOKEN is not set.</dd>
+
+ <dt>ATMOS_BITBUCKET_USERNAME</dt>
+ <dd>Bitbucket username for authentication. Takes precedence over BITBUCKET_USERNAME.</dd>
+
+ <dt>BITBUCKET_USERNAME</dt>
+ <dd>Used as a fallback when ATMOS_BITBUCKET_USERNAME is not set. Bitbucket requires a valid username and does not accept dummy values like `"x-access-token"`.</dd>
+</dl>
```

Comment:
```suggestion
<dl>
 <dt>`ATMOS_BITBUCKET_TOKEN`</dt>
 <dd>Bitbucket app password for API requests; used to avoid rate limits. When both `ATMOS_BITBUCKET_TOKEN` and `BITBUCKET_TOKEN` are defined, the former prevails.</dd>

 <dt>`BITBUCKET_TOKEN`</dt>
 <dd>Used as a fallback when `ATMOS_BITBUCKET_TOKEN` is not set.</dd>

 <dt>`ATMOS_BITBUCKET_USERNAME`</dt>
 <dd>Bitbucket username for authentication. Takes precedence over `BITBUCKET_USERNAME`.</dd>

 <dt>`BITBUCKET_USERNAME`</dt>
 <dd>Used as a fallback when `ATMOS_BITBUCKET_USERNAME` is not set. Bitbucket requires a valid username and does not accept dummy values like `x-access-token`.</dd>
</dl>
```